src/telemetry/dto/write-peer-list.dto.ts

```diff
@@ -0,0 +1,31 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+import { Type } from 'class-transformer';
+import {
+  ArrayMaxSize,
+  IsArray,
+  IsDate,
+  IsNotEmpty,
+  IsString,
+  MaxLength,
+} from 'class-validator';
+
+export class WritePeerListDto {
+  @IsArray()
+  @ArrayMaxSize(10000)
+  @MaxLength(100, {
+    each: true,
+  })
+  readonly peers!: string[];
+
+  @IsString()
+  @IsNotEmpty()
+  @MaxLength(100)
+  readonly nodeId!: string;
```

Comment:
We use snake case for our API payloads

```suggestion
  readonly node_id!: string;
```